quickstart/configs.go

```diff
@@ -22,15 +24,22 @@ func configBaseForm(quickstart *Quickstart) (*State, error) {
 		configFields := []huh.Field{
 			huh.NewInput().
 				Title("Choose an sdkClassName for your target:").
+				Placeholder("idiomatic name for your SDK object").
 				Inline(true).
 				Prompt(" ").
 				Value(&sdkClassName),
 		}
-		configFields = append(configFields, languageSpecificForms(target.Target)...)
-		if _, err := tea.NewProgram(charm.NewForm(huh.NewForm(
+		languageForms, err := languageSpecificForms(target.Target)
+		if err != nil {
+			return nil, err
+		}
+
+		configFields = append(configFields, languageForms...)
+		form := huh.NewForm(
 			huh.NewGroup(
 				configFields...,
-			)),
+			))
+		if _, err := tea.NewProgram(charm.NewForm(form,
 			fmt.Sprintf("Let's setup a gen.yaml config for your target %s of type %s", key, target.Target),
```

Comment:
At this point the user has no idea what "gen.yaml" is. Maybe I'd change this to `Let's configure your Go target (my-first-target)`